test/e2e/base/base.go

```diff
@@ -0,0 +1,21 @@
+package base
+
+import (
+	"time"
+
+	"github.com/go-resty/resty/v2"
+)
+
+const (
+	DefaultAddress = "127.0.0.1:12379"
```

Comment:
Be strict. Always follow the rules:

1. Do you really need to capitalize the first letter?
2. If so, please add some comments for it.

Reply:
added

Reply:
OK.